src/test/java/com/puppycrawl/tools/checkstyle/grammar/comments/AllSinglelineCommentsTest.java

```diff
@@ -48,12 +47,10 @@ protected String getPackageLocation() {
 
     @Test
     public void testAllSinglelineComments() throws Exception {
-        final DefaultConfiguration checkConfig =
-            createModuleConfig(SinglelineCommentListenerCheck.class);
         final String path = getPath("InputFullOfSinglelineComments.java");
         lineSeparator = CheckUtil.getLineSeparatorForFile(path, StandardCharsets.UTF_8);
         final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
-        verify(checkConfig, path, expected);
+        verifyWithInlineConfigParser(path, expected);
```

Comment:
This becomes weird. We try to verify.... But in reality it is not verication, it  just execution of module. I think we need new method `execute` that just run module and do not validation.
Update in input file become not required.

Same concept we will use in case we put exceptional values in config of module.
So test expects exception.